build.gradle

```diff
@@ -1,5 +1,5 @@
 plugins {
     id 'com.android.application' version '7.4.0' apply false
     id 'com.android.library' version '7.4.0' apply false
```

Comment:
Doesn't seem be required, we don't have any submodules, only the main "app"

Reply:
I can handle it in next PR